src/components/Conference.tsx

```
@@ -152,7 +152,11 @@ export class Conference extends React.Component<IConferenceProps, IConferenceSta
             return;
         };
 
-        this.connection = this.props.connect();
+        const connection = this.props.connect();
+        if (!connection) {
+            return
```

Shall we throw an error here?

---

Done